examples/megacli.cpp

```diff
@@ -2500,6 +2500,35 @@ void exec_treecompare(autocomplete::ACState& s);
 void exec_querytransferquota(autocomplete::ACState& s);
 
 
+void exec_showattributes(autocomplete::ACState& s)
+{
+    if (Node* n = nodeFromRemotePath(s.words[1].s))
```

Comment:
Might be more expressive to name it `node` instead of `n`? Also should be declared const: `const Node* const node`

Reply:
Technically `Node const * const`  would be more correct - however making pointers const is unusual and distracting - I've updated it to just `const Node*` :-)